### ISSUE TITLE
Table doesn't render properly until createSpreadsheetData is used

### ISSUE THREAD

**Issue by ottosson** [handsontable/handsontable#10089]
### Description
I'm not sure if this is me doing things wrong, but I get a super strange issue where the table doesn't render column functions (sorting, filtering, auto size for columns and potentially other things). However when I run `Handsontable.helper.createSpreadsheetData(...);` the table renders properly. I don't know if I'm missing things, but one: Why does this ever happen, and two: Why does createSpreadSheetData help mitigate the issue? I've looked at the code for createSpreadSheetData and it doesn't do anything magic, it's just a function without side effects.

Feels like a React-related issue for sure.

### Steps to reproduce

1. create a new grid
2. activate column features, like sorting, filtering, autosize
3. populate grid with data
4. column features doesn't work
5. run Handsontable.helper.createSpreadsheetData(anyNumber, anyNumber)

### Demo
https://codesandbox.io/s/handsontable-react-data-grid-hello-world-app-forked-ldo0j4?file=/src/index.tsx
Uncomment line 32 to mitigate the issue.

### Your environment
* React wrapper version: 12.2.0
* Handsontable version: 12.2.0
* Browser name and version: Chrome+Firefox (latest)
* Operating system: MacOS

**Comment by ottosson** [comment]
I don't think it's Code sandbox problem in this case. I experience it in my local project.

But yes, that seems to have been the problem. I needed to register the plugins. It's new stuff for me. Has been a while since I used HoT.

What's the reason for having to register modules manually? Performance? Where can I read more about registering modules?

Still though, out of curiosity: why does it fix the problem using createSpreadsheetData?
